Themes/default/css/index.css

```diff
@@ -2794,7 +2798,7 @@ dl.addrules dt.floatleft {
 	background: #f0f4f7;
 }
 .windowbg:nth-of-type(odd) {
-	background: #fdfdfd; /* needs new color */
+	background: #EEEEEE;
```

Comment:
Too whiteish? I don't know, I asked for opinions in topic (private board), but none said anything. Let me revert back, we can discuss it later on (just a one line).